common/observer/dispatcher/direct_event_dispatcher.go

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
I want to know why `observer` is set in `common`. Who will use it.

Reply:
This observer will dispatcher  all event will change service status such as service create event, shutdown event, config exported event etc